docs/user_guide/ttl.md

```diff
@@ -17,11 +17,42 @@ minimum, not maximum, TTL. It is also advisable to schedule the periodic purging
 in case of infra delays or failures.
 
 ## Usage
+There are two ways to achieve TTL compliance, via [Repush with TTL](#repush-with-ttl) or via [empty push](#empty-push).
+The major differences between both approaches is the original data source and how real-time buffer is replayed.
 
-There are two ways to achieve TTL compliance, via [empty push](#empty-push) or via [Repush with TTL](#experimental-repush-with-ttl).
+|                 | Source Data for TTL    | Real-time buffer replay                                               |
+|-----------------|------------------------|-----------------------------------------------------------------------|
+| Empty push      | None                   | Replay real-time buffer with `hybrid-rewind-seconds` config           |
+| Repush with TTL | Existing version topic | Replay real-time buffer with `rewind.time.in.seconds.override` config |
 
-## Empty push
+## Repush with TTL
```

Comment:
Thank you for updating the doc as well!  Curious why are we allowing the store rewind time and ttl period to be separately configured ? If for compliance then the rewind time would also cause problem ?

Reply:
Why does TTL period need to be configured through "rewind time"? What if use cases want 1 year of TTL? GDPR compliance would only allow a max of 30 days. So, that restricts repush from supporting 1 year TTLs because we only use the same store config for TTL and topic retention. This config allows us to disambiguate the two things.